src/css/index.scss

```diff
@@ -1,2 +1,167 @@
 @import './global';
 @import './information';
+
+$basic-blue: #bcc8dc;
+$basic-grey: #edeff5;
+$dark-red: #d93347;
+$dark-blue: #314a74;
+
+.HOME {
+  position: fixed;
```

Comment:
The fixed positioning should better be applied to the wrapper element of home and download links, since both of them needs to be fixed.